graph.js

```diff
@@ -133,17 +152,39 @@ app
     }
   })
 
+/**
+ * graph auth */
+app
+  .command('auth [NODE] [AUTH_TOKEN]')
+  .description("Stores auth token on the system's keychain for use by deploy")
+  .action((node, authToken) => {
+    if (
+      authToken === undefined ||
+      node === undefined ||
+      authToken > 200
```

Comment:
This should be `authToken.length`, I think?

Please rename `AUTH_TOKEN` and `authToken` to `ACCESS_TOKEN` and `accessToken`.

Also: `Stores the auth token on the system's keychain for use by deploy` is not the most user-friendly message. How about this?
```js
.description("Sets the access token to use when deploying to a Graph node")
```